<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>792</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0">
     <widget class="QFrame" name="frame_2">
      <property name="maximumSize">
       <size>
        <width>300</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
	border-radius: 10px;
	color: rgb(255, 255, 255);
background-color: qradialgradient(cx:0.1, cy:0.1, radius:0.5, fr:0.5, fy:0.5, stop:o rgb(0,170, 247),stop:1 rgb(60, 1,252));</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QComboBox" name="comboBox">
           <property name="styleSheet">
            <string notr="true">font: 13pt &quot;黑体&quot;;
color: rgb(255, 255, 255);
background-color: transparent;
padding:12px;</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QPushButton" name="pushButton">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>25</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-radius: 10px;
background-color:qradialgradient(cx:0.1, cy:0.1, radius:0.5, fr:2, fy:0.5, stop:o rgb(0,190, 247),stop:1 rgb(60, 160,252));
coler: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>自适应</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="styleSheet">
            <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 150, 0, 255), stop:1 rgba(255, 190, 0, 255));
border-radius: 10px;</string>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 13pt &quot;黑体&quot;;
background-color: transparent;</string>
           </property>
           <property name="text">
            <string>空域图像处理</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>125</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QListWidget {
	border-radius: 10px;
	color: rgb(255, 255, 255);
	background-color:transparent
}
QListWidget::item{
background-color: transparent;
height:25px;
padding-left:12px;
}
/**列表项扫过*/
QListWidget::item:hover {
background-color: rgba(255,255,255,50);
}
/**列表项选中*/
QListWidget::item:selected {
/**background-calor: transparent;*/
background-color: rgb(220,220,220);
color: rgb(60, 60, 60);
}
color: rgb(255, 255, 255);
</string>
           </property>
           <property name="layoutMode">
            <enum>QListView::Batched</enum>
           </property>
           <item>
            <property name="text">
             <string>对比度</string>
            </property>
            <property name="icon">
             <iconset resource="../icon.qrc">
              <normaloff>:/solid/fontawesome-free-6.2.1-desktop/fontawesome-free-6.2.1-desktop/svgs/solid/circle-dot.svg</normaloff>:/solid/fontawesome-free-6.2.1-desktop/fontawesome-free-6.2.1-desktop/svgs/solid/circle-dot.svg</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>平滑及锐化</string>
            </property>
            <property name="icon">
             <iconset resource="../icon.qrc">
              <normaloff>:/solid/fontawesome-free-6.2.1-desktop/fontawesome-free-6.2.1-desktop/svgs/solid/circle-dot.svg</normaloff>:/solid/fontawesome-free-6.2.1-desktop/fontawesome-free-6.2.1-desktop/svgs/solid/circle-dot.svg</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>图形学</string>
            </property>
            <property name="icon">
             <iconset resource="../icon.qrc">
              <normaloff>:/solid/fontawesome-free-6.2.1-desktop/fontawesome-free-6.2.1-desktop/svgs/solid/circle-dot.svg</normaloff>:/solid/fontawesome-free-6.2.1-desktop/fontawesome-free-6.2.1-desktop/svgs/solid/circle-dot.svg</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>加噪</string>
            </property>
            <property name="icon">
             <iconset resource="../icon.qrc">
              <normaloff>:/solid/fontawesome-free-6.2.1-desktop/fontawesome-free-6.2.1-desktop/svgs/solid/circle-dot.svg</normaloff>:/solid/fontawesome-free-6.2.1-desktop/fontawesome-free-6.2.1-desktop/svgs/solid/circle-dot.svg</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>边缘提取</string>
            </property>
            <property name="icon">
             <iconset resource="../icon.qrc">
              <normaloff>:/solid/fontawesome-free-6.2.1-desktop/fontawesome-free-6.2.1-desktop/svgs/solid/circle-dot.svg</normaloff>:/solid/fontawesome-free-6.2.1-desktop/fontawesome-free-6.2.1-desktop/svgs/solid/circle-dot.svg</iconset>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 13pt &quot;黑体&quot;;
background-color: transparent;</string>
           </property>
           <property name="text">
            <string>频域图像处理</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget_2">
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QListWidget {
	border-radius: 10px;
	color: rgb(255, 255, 255);
	background-color: transparent;
}
QListWidget::item{
background-color: transparent;
height:25px;
padding-left:12px;
}
/**列表项扫过*/
QListWidget::item:hover {
background-color: rgba(255,255,255,50);
}
/**列表项选中*/
QListWidget::item:selected {
/**background-calor: transparent;*/
background-color: rgb(220,220,220);
color: rgb(60, 60, 60);
}
</string>
           </property>
           <item>
            <property name="text">
             <string>滤波</string>
            </property>
            <property name="icon">
             <iconset resource="../icon.qrc">
              <normaloff>:/solid/fontawesome-free-6.2.1-desktop/fontawesome-free-6.2.1-desktop/svgs/solid/circle-dot.svg</normaloff>:/solid/fontawesome-free-6.2.1-desktop/fontawesome-free-6.2.1-desktop/svgs/solid/circle-dot.svg</iconset>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 13pt &quot;黑体&quot;;
background-color: transparent;</string>
           </property>
           <property name="text">
            <string>二值化图像形态学</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget_3">
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>25</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QListWidget {
	border-radius: 10px;
	color: rgb(255, 255, 255);
	background-color: transparent;
}
QListWidget::item{
background-color: transparent;
height:25px;
padding-left:12px;
}
/**列表项扫过*/
QListWidget::item:hover {
background-color: rgba(255,255,255,50);
}
/**列表项选中*/
QListWidget::item:selected {
/**background-calor: transparent;*/
background-color: rgb(220,220,220);
color: rgb(60, 60, 60);
}
</string>
           </property>
           <item>
            <property name="text">
             <string>图像二值化</string>
            </property>
            <property name="icon">
             <iconset resource="../icon.qrc">
              <normaloff>:/solid/fontawesome-free-6.2.1-desktop/fontawesome-free-6.2.1-desktop/svgs/solid/circle-dot.svg</normaloff>:/solid/fontawesome-free-6.2.1-desktop/fontawesome-free-6.2.1-desktop/svgs/solid/circle-dot.svg</iconset>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">font: 13pt &quot;黑体&quot;;
background-color: transparent;</string>
           </property>
           <property name="text">
            <string>数据标签处理</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget_4">
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QListWidget {
	border-radius: 10px;
	color: rgb(255, 255, 255);
	background-color: transparent;
}
QListWidget::item{
background-color: transparent;
height:25px;
padding-left:12px;
}
/**列表项扫过*/
QListWidget::item:hover {
background-color: rgba(255,255,255,50);
}
/**列表项选中*/
QListWidget::item:selected {
/**background-calor: transparent;*/
background-color: rgb(220,220,220);
color: rgb(60, 60, 60);
}
</string>
           </property>
           <item>
            <property name="text">
             <string>抠图(前景背景分离)</string>
            </property>
            <property name="icon">
             <iconset resource="../icon.qrc">
              <normaloff>:/solid/fontawesome-free-6.2.1-desktop/fontawesome-free-6.2.1-desktop/svgs/solid/circle-dot.svg</normaloff>:/solid/fontawesome-free-6.2.1-desktop/fontawesome-free-6.2.1-desktop/svgs/solid/circle-dot.svg</iconset>
            </property>
           </item>
           <item>
            <property name="text">
             <string>数据增强</string>
            </property>
            <property name="icon">
             <iconset resource="../icon.qrc">
              <normaloff>:/solid/fontawesome-free-6.2.1-desktop/fontawesome-free-6.2.1-desktop/svgs/solid/circle-dot.svg</normaloff>:/solid/fontawesome-free-6.2.1-desktop/fontawesome-free-6.2.1-desktop/svgs/solid/circle-dot.svg</iconset>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QGraphicsView" name="graphicsView">
           <property name="cursor" stdset="0">
            <cursorShape>CrossCursor</cursorShape>
           </property>
           <property name="renderHints">
            <set>QPainter::Antialiasing|QPainter::HighQualityAntialiasing|QPainter::LosslessImageRendering|QPainter::SmoothPixmapTransform|QPainter::TextAntialiasing</set>
           </property>
           <property name="dragMode">
            <enum>QGraphicsView::ScrollHandDrag</enum>
           </property>
           <property name="transformationAnchor">
            <enum>QGraphicsView::AnchorUnderMouse</enum>
           </property>
           <property name="resizeAnchor">
            <enum>QGraphicsView::AnchorUnderMouse</enum>
           </property>
           <property name="viewportUpdateMode">
            <enum>QGraphicsView::FullViewportUpdate</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGraphicsView" name="graphicsView_2">
           <property name="cursor" stdset="0">
            <cursorShape>CrossCursor</cursorShape>
           </property>
           <property name="renderHints">
            <set>QPainter::Antialiasing|QPainter::HighQualityAntialiasing|QPainter::LosslessImageRendering|QPainter::SmoothPixmapTransform|QPainter::TextAntialiasing</set>
           </property>
           <property name="dragMode">
            <enum>QGraphicsView::ScrollHandDrag</enum>
           </property>
           <property name="transformationAnchor">
            <enum>QGraphicsView::AnchorUnderMouse</enum>
           </property>
           <property name="resizeAnchor">
            <enum>QGraphicsView::AnchorUnderMouse</enum>
           </property>
           <property name="viewportUpdateMode">
            <enum>QGraphicsView::FullViewportUpdate</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuimport">
     <property name="title">
      <string>import</string>
     </property>
     <addaction name="actionimport_img"/>
     <addaction name="actionopen_data_dir"/>
    </widget>
    <widget class="QMenu" name="menuexport">
     <property name="title">
      <string>export</string>
     </property>
     <addaction name="actionexport_as_png"/>
    </widget>
    <addaction name="menuimport"/>
    <addaction name="menuexport"/>
    <addaction name="actionsave"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionopen_data_dir"/>
   <addaction name="actionimport_img"/>
  </widget>
  <action name="actionimport_img">
   <property name="icon">
    <iconset resource="../icon.qrc">
     <normaloff>:/solid/fontawesome-free-6.2.1-desktop/fontawesome-free-6.2.1-desktop/svgs/solid/file-import.svg</normaloff>:/solid/fontawesome-free-6.2.1-desktop/fontawesome-free-6.2.1-desktop/svgs/solid/file-import.svg</iconset>
   </property>
   <property name="text">
    <string>import img</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionexport_as_png">
   <property name="icon">
    <iconset resource="../icon.qrc">
     <normaloff>:/solid/fontawesome-free-6.2.1-desktop/fontawesome-free-6.2.1-desktop/svgs/solid/file-export.svg</normaloff>:/solid/fontawesome-free-6.2.1-desktop/fontawesome-free-6.2.1-desktop/svgs/solid/file-export.svg</iconset>
   </property>
   <property name="text">
    <string>export as png</string>
   </property>
   <property name="statusTip">
    <string>export as png</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionopen_data_dir">
   <property name="icon">
    <iconset resource="../icon.qrc">
     <normaloff>:/solid/fontawesome-free-6.2.1-desktop/fontawesome-free-6.2.1-desktop/svgs/solid/list-ul.svg</normaloff>:/solid/fontawesome-free-6.2.1-desktop/fontawesome-free-6.2.1-desktop/svgs/solid/list-ul.svg</iconset>
   </property>
   <property name="text">
    <string>open data dir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionsave">
   <property name="icon">
    <iconset resource="../icon.qrc">
     <normaloff>:/solid/fontawesome-free-6.2.1-desktop/fontawesome-free-6.2.1-desktop/svgs/solid/file.svg</normaloff>:/solid/fontawesome-free-6.2.1-desktop/fontawesome-free-6.2.1-desktop/svgs/solid/file.svg</iconset>
   </property>
   <property name="text">
    <string>save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../icon.qrc"/>
 </resources>
 <connections/>
</ui>
